### PR TITLE
Update quickstart-integratedsearchdisk.md

### PR DESCRIPTION
**Pull request by whenov**

### CODE REVIEW

File: doc/quickstart-integratedsearchdisk.md [hunk 1]
```diff
@@ -302,7 +302,7 @@ public class IndexingAndRetrievalExample {
 			int docid = doc.getDocid();
 			double score = doc.getScore();
 			String docno = doc.getMetadata("docno")
-			System.out.println("   Rank "+i+": "+docid+" "+docno+" "+score);
```
Comment (cmacdonald) [inline doc/quickstart-integratedsearchdisk.md:305]: Thanks for your efforts.

Any reason why you removed remove the rank here?

Comment (whenov) [reply doc/quickstart-integratedsearchdisk.md:305]: Because the `i` variable doesn't actually exist here?

Comment (cmacdonald) [reply doc/quickstart-integratedsearchdisk.md:305]: Could you restore it?

Comment (whenov) [reply doc/quickstart-integratedsearchdisk.md:305]: It is restored now.

Comment (cmacdonald) [reply doc/quickstart-integratedsearchdisk.md:305]: I have made it work, but adding the i (now called rank) variable and updating it in the iterator. Do you agree?

Comment (whenov) [reply doc/quickstart-integratedsearchdisk.md:305]: Looks good to me, thanks!

**Pull request merged**